lib/AdapterRpc.php

```diff
@@ -480,4 +480,46 @@ public function getMemberStatusByUserAndVo($user, $vo)
         }
         return $member->getStatus();
     }
+
+    public function getResourceCapabilities($entityId, $userGroups)
+    {
+        $facility = $this->connector->get('facilitiesManager', 'getFacilitiesByAttribute', [
```

Comment:
Because the method getFacilitiesByAttribute can return more values than one, you should add check if you get one or more results.